package.json

```diff
@@ -28,6 +28,7 @@
     "chrome-devtools-frontend": "1.0.381789",
     "chrome-remote-interface": "^0.11.0",
     "devtools-timeline-model": "1.0.16",
+    "jsdom": "^8.1.0",
```

Comment:
https://github.com/samccone/engine-deps

If we want to support older versions of node....